lib/mintel/alerts/fluentd.libsonnet

```diff
@@ -0,0 +1,24 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'fluentd.alerts',
+        rules: [
+          {
+            alert: 'FluentdHighOutputRetry',
+            annotations: {
+              description: 'Fluentd number of output retry has been increasing for 1h',
+              summary: 'Fluentd number of output retry has been increasing for 1h',
+              runbook_url: '%(runBookBaseURL)s/core/FLuentdHighOutputRetry.md' % $._config,
```

Comment:
Should this reference `FluentdHighOutputRetry.md` instead of `FLuentdHighOutputRetry.md` ? (capital L in fluent)